_search-plugins/user-behavior-insights/index.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+title: User Behavior Insights
+parent: Search relevance
+has_children: false
+nav_order: 220
+---
+
+# User behavior insights
+
+User Behavior Insights, or UBI, is a community plugin for capturing client-side events and queries for the purposes of improving search relevance and user experience. UBI consists of three components:
+
+* An OpenSearch [plugin](https://github.com/o19s/opensearch-ubi) that facilitates the storage of client-side events and queries.
+* A client-side JavaScript library that demonstrates how to capture events and send those events to the OpenSearch UBI plugin.
+* An OpenSearch Dashboards plugin that provides access to the collected events and queries for analysis. This component is still largely in the design phase.
+
+UBI is currently only supported in OpenSearch 2.12.0.
+{: .warning }
+
+## Installing the plugin
+
+The User Behavior Insights plugin is available for OpenSearch 2.12.0. Run the following command to install the plugin.
+
+````bash
+./bin/opensearch-plugin install \
+   "https://github.com/o19s/opensearch-ubi/releases/download/0.0.8/opensearch-ubi-0.0.8-os2.12.0.zip"
+````
+
+After installing the UBI plugin you can find [documentation](https://github.com/o19s/opensearch-ubi/blob/main/documentation.md).
+
+## Using the plugin
```

Comment:
I would remove this heading and the intro sentence and just make the following headings level 2 instead of level 3.